iina/Base.lproj/MiniPlayerWindowController.xib

```diff
@@ -112,7 +112,7 @@
                                     <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="h25-bp-EvL" customClass="ScrollingTextField" customModule="IINA" customModuleProvider="target">
                                         <rect key="frame" x="133" y="26" width="35" height="16"/>
                                         <textFieldCell key="cell" lineBreakMode="truncatingMiddle" sendsActionOnEndEditing="YES" alignment="center" title="Title" id="Rs9-6l-6vp">
-                                            <font key="font" metaFont="system"/>
```

Comment:
I assume Xcode did this automatically, but I am not sure it's what we want…

Reply:
Unfortunately Xcode is doing this all the time, every time when you have changes in a xib file. But I don't think it will make a difference, otherwise Xcode shouldn't do this automatically,